src/vector-crypto.adoc

```diff
@@ -308,6 +313,18 @@ all other `SEW` values are _reserved_.
 
 |===
 
+EMUL and Source/Destination index alignment constraints::
++
+A *Vector Element Group* operand or destination has `EMUL = LMUL`.
++
+A *Scalar Element Group* operand or destination has `EMUL = ceil(EGW / VLEN)`.
++
+The https://github.com/riscv/riscv-v-spec/blob/master/v-spec.adoc#342-vector-register-grouping-vlmul20[Standard RVV vector register group alignment constraints] apply.
```

Comment:
We shouldn't reference the `riscv-v-spec` repo anymore.  Since the alignment constraints section is now part of the same spec as this--both are in the unpriv ISA manual--we can use an asciidoc reference rather than a URL.  I guess we can fix the other reference to `riscv-v-spec` while we are at it.

Reply:
Should be corrected in https://github.com/riscv/riscv-isa-manual/pull/1653/commits/2161a3b88338ea04a30ceec06149b9008e09cc0e